kubecost-cloud-request-right-sizing.md

```diff
@@ -0,0 +1,26 @@
+# Kubecost Cloud: Request Right-Sizing Recommendations
+
+{% hint style="info" %}
+This documentation should only be consulted when using Kubecost Cloud! For information about request right-sizing recommendations for self-hosted Kubecost, see [here](https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/savings/auto-request-sizing).
+{% endhint %}
+
+Kubecost Cloud is able to provide recommendations for right-sizing for your container requests to ensure they are as cost-effective as possible. Recommendations are provided for all namespaces within your cluster.
```

Comment:
```suggestion
Kubecost Cloud is able to provide recommendations for right-sizing your container requests to ensure they are as cost-effective as possible. Recommendations are provided for all namespaces within your cluster.
```

Reply:
Can right-sizing be used as a verb in this instance? Or is right-sizing a process/noun in any context?

Reply:
Sounds correct to me to use it as a verb in this instance. Up to you though, should most importantly maintain consistency across all references